utils.ts

```diff
@@ -297,19 +297,19 @@ export const checkIfResultsAreSatisfactory = (
   const PREFERRED_LAYERS = ['venue', 'address', 'street', 'intersection']
 
   // Check for zero length
-  if (features.length === 0) return false
+  if (features?.length === 0) return false
```

Comment:
If `features` is null does it continue execution? I would include `!features` in this `if` condition.

Reply:
Looks like we should be alright! <img width="190" alt="Screen Shot 2022-08-17 at 9 37 11 AM" src="https://user-images.githubusercontent.com/86619099/185148149-577223c7-c8bf-4a09-a93e-34ca114f7bf0.png">